django_keycloak/mixins.py

```diff
@@ -33,11 +33,12 @@ def tearDown(self):  # pylint: disable=invalid-name
         self.keycloak_cleanup()
```

Comment:
Since the setUp and tearDown overloading has been deprecated, we might as well remove it now

Reply:
Why is `setUp` and `tearDown` deprecated? Did django deprecated them?

Reply:
They are not deprecated, but the method how the mixin integrated itself was a bit janky by implementing `setUp` / `tearDown` itself. It depended on the order it was included and might not call other `setUp` / `tearDown` methods.

A more robust method that also gives more flexibility is requiring the user to call `keycloak_init` and `keycloak_cleanup` in the test class `setUp` / `tearDown` methods.

Reply:
something like this:
```python
database = []


class MyMixin:
    def __init__(self) -> None:
        database.append([1, 2, 3])

    def __del__(self) -> None:
        while database:
            database.pop()


class MyClass(MyMixin):
    def test_something(self):
        pass


a = MyClass()

print("database: ", database) # --> database:  [[1, 2, 3]]
del a
print("database: ", database) # --> database:  []
```

Reply:
I wouldn't use del as it can be called at [any point by the garbage collector](https://stackoverflow.com/a/1481512/6783666). Also, it removes the flexibility of allowing code to be added before or after the init and cleanup methods.